### PR TITLE
Microsoft Student Developer Day program launch.

### PR DESCRIPTION
**Pull request by vivsridh4**
The first MSD Day event is launched and this PR has the details.

### CODE REVIEW

File: microsoft-student-developer-day/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+Microsoft Student Developer Day is held once a month at Microsoft Reactor India to empower & encourage students to share learning with the community via reactor platform.
```
Comment (liamchampton) [inline microsoft-student-developer-day/README.md:1]: `via reactor platform` -> `via the Reactor platform`
OR
`via reactor platform` -> `via Reactor platforms`

**Review by liamchampton — COMMENTED** [review]
Small grammar suggestion as it doesn't read correctly to me - otherwise looks good

**Comment by vivsridh4** [comment]
@liamchampton can you review now and merge.

**Review by liamchampton — APPROVED** [review]
lgtm

**Pull request merged**